openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/groups/Members.java

```diff
@@ -0,0 +1,159 @@
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.security.AuthorityService;
+import org.alfresco.service.cmr.security.AuthorityType;
+import org.alfresco.service.cmr.security.PersonService;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.extensions.webscripts.*;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Members extends AbstractWebScript {
+
+    private AuthorityService authorityService;
+    private PersonService personService;
+    // Logger
+    private static final Log logger = LogFactory.getLog(Members.class);
+
+    @Override
+    public void execute(WebScriptRequest req, WebScriptResponse res) throws IOException {
+        String method = req.getServiceMatch().getWebScript().getDescription().getMethod();
+        Map<String, String> templateArgs = req.getServiceMatch().getTemplateVars();
+        String shortName = templateArgs.get("shortName");
+        JSONObject parsedRequest;
+        // Parse the JSON, if supplied
+        JSONParser parser = new JSONParser();
+        try {
+            parsedRequest = (JSONObject) parser.parse(req.getContent().getContent());
+        } catch (IOException | ParseException io) {
+            throw new WebScriptException(Status.STATUS_BAD_REQUEST, "Invalid JSON: " + io.getMessage());
+        }
+        //The shortName is not blank
+        if (StringUtils.isBlank(shortName)) {
+            throw new WebScriptException("The shortName can't be empty/blank");
+        }
+        //Check if the group exists before trying to perform the operation
+        if (!authorityService.authorityExists(authorityService.getName(AuthorityType.GROUP, shortName)))
+            throw new WebScriptException("The group (" + shortName + ") does not/ no longer exists");
+
+        try {
+            switch (method) {
+                case "PUT":
+                    addMembers(req, res, shortName, parsedRequest);
+                    break;
+            }
+        } catch (JSONException e) {
+            throw new WebScriptException(Status.STATUS_BAD_REQUEST, "Invalid JSON: " + e.getMessage());
+        }
+    }
+
+    private void addMembers(WebScriptRequest req, WebScriptResponse res, String groupShortName, JSONObject parsedRequest) throws IOException, JSONException {
+        String users = getOrNull(parsedRequest, "users");
+        String groups = getOrNull(parsedRequest, "groups");
+        StringBuffer failed = new StringBuffer();
+        String groupFullName ="GROUP_"+ groupShortName;
+
+        if (StringUtils.isNotBlank(users)) {
+            try {
+                List<String> candidates = Arrays.asList(StringUtils.split(users, ","));
```

Comment:
have you considered parsing this param to JSONArray?
and on current situation it is not necessary to convert to list, you can iterate over array